contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -54,15 +66,16 @@ select
   , 0 as parent_object_id
   , 'V'::varchar(2) as type 
   , 'VIEW'::varchar(60) as type_desc
-  , null::timestamp as create_date
-  , null::timestamp as modify_date
+  , vd.create_date::timestamp as create_date
+  , vd.create_date::timestamp as modify_date
```

Comment:
The view sys.views currently has create_date/modify_date as timestamp and it will require rename-recreate method to fix it so I am thinking to target that as a different change and keep this PR focused to support the required functionality.